default/scripting/policies/EXOBOT_PRODUCTIVITY.focs.txt

```diff
@@ -19,9 +19,10 @@ Policy
                 TargetIndustry low = 0
                 Focus type = "FOCUS_INDUSTRY"
             ]
+            // Very bad multiplier is applied before, so it must be reverted before applying bad multiplier.
             effects = SetTargetIndustry value = Value
                         * (NamedReal name = "PLC_EXOBOT_PRODUCTIVITY_TARGET_INDUSTRY_SCALING"
-                                     value = [[BAD_MULTIPLIER]] - [[VERY_BAD_MULTIPLIER]])
+                                     value = (1 / [[VERY_BAD_MULTIPLIER]]) * [[BAD_MULTIPLIER]])
```

Comment:
Can this just be
````
            effects = SetTargetIndustry value = Value
                        * (NamedReal name = "PLC_EXOBOT_PRODUCTIVITY_TARGET_INDUSTRY_SCALING"
                                     value = [[BAD_MULTIPLIER]] / [[VERY_BAD_MULTIPLIER]])
````

Reply:
Of course, don't know why I did not write it like that in the first place.
Does it help when I suggest the change here?
```suggestion
                                     value = ([[BAD_MULTIPLIER]] / [[VERY_BAD_MULTIPLIER]])
```

Reply:
seemingly yes, there is a button